CSS/style.css

```diff
@@ -1,8 +1,8 @@
 /* Pre style */
 
 body {
-    font-family: sans-serif;
-    background-color: #f8faff;
+  font-family: sans-serif;
+  background-color: #f8faff;
 }
 
```

Comment:
Css file should contain one be linked to one class of html file that contain header not element